optaplanner-examples/src/main/java/org/optaplanner/examples/conferencescheduling/domain/Room.java

```diff
@@ -93,6 +93,16 @@ public void setTagSet(Set<String> tagSet) {
     // With methods
     // ************************************************************************
 
+    public Room withCapacity(int capacity) {
+        this.capacity = capacity;
+        return this;
+    }
+
+    public Room withTagSet(Set<String> tagSet) {
+        this.tagSet = tagSet;
+        return this;
+    }
```

Comment:
Nitpick: ideally the with methods are in the same order as the fields (and the getter/setters), so tagSet after withUnavailableTimeslotSet. It's just a bit nicer to read, and some OCD example users love such detail :)

Reply:
Btw, a bit below in this class, there is a parameter named unavailableTimeslotTest incorrectly.